spec/feature/authenticated_user_spec.rb

```diff
@@ -309,40 +310,40 @@
     expect(page).to_not have_content("Remove from Category")
   end
 
-  xit "cannot create a category" do
+  it "cannot create a category" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                     and_return(valid_user)
     visit new_admin_category_path
     expect(page).to have_content("Page Not Found")
   end
 
-  xit "cannot modify a category" do
+  it "cannot modify a category" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                     and_return(valid_user)
     visit edit_admin_category_path(category1)
     expect(page).to have_content("Page Not Found")
   end
 
-  xit "cannot make themselves an admin" do
+  it "cannot make themselves an admin" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                     and_return(valid_user)
     visit new_admin_path
     expect(page).to have_content("Page Not Found")
   end
-
-  def click_add_to_cart_link(category)
-    click_link_or_button "Menu"
-    within(".categories") do
-      within("div##{category}") do
-          click_link "Add to Cart"
-      end
-    end
-  end
-
+  #
+  # def click_add_to_cart_link(category)
+  #   click_link_or_button "Menu"
+  #   within(".categories") do
+  #     within("div##{category}") do
+  #         click_link "Add to Cart"
+  #     end
+  #   end
+  # end
+  #
   def valid_user_logs_in
     click_link "Log In"
     fill_in "session_email", with: "alice@gmail.com"
     fill_in "session_password", with: "password"
     click_button "Log In"
   end
-end
+ end
```

Comment:
`end` at 349, 1 is not aligned with `describe "an authenticated user" do` at 3, 0